benchmarks/README.md

```diff
@@ -3,5 +3,15 @@
 ## To run the benchmark use
 
 ```
-$ ./gradlew :benchmarks:jmh
+$ ./gradlew :opencensus-benchmarks:jmh
 ```
+
+## To debug compilation errors
+When you make incompatible changes in the Benchmarks classes you may get compilation errors which
+are related to the old code not being compatible with the new code. Run this commands to clean 
+the Gradle's cache:
```

Comment:
Why doesn't Gradle detect that it needs to recompile?

Reply:
Few reasons that I found on internet:
1) Any plugin cannot delete the generated code (jmh generates code) because if the user configured the directory as the same as source code the plugin will delete users source code.
2) For some reasons which I couldn't find the right answer (or I did not understand exactly) after you run jmh  a demon will stay alive which may cache the  generated code in memory and generates the same code even if the files were changed (I am not clear why this happens because they should use some hash algorithm to determine if the file changed).

Reply:
I think it would help to give a little more detail about the reason in the readme.

Reply:
Done.